### DISCUSSION TITLE
Are there any recent "Good First Issues"?

### DISCUSSION THREAD

**Discussion by Shadow53** [GothenburgBitFactory/taskwarrior#3076] [Q&A]
The most recent issue tagged "good first issue" is from 2021, with the most recent open one being from 2019.

Are there any more recent open issues that would be good for new contributors? If so, can they get tagged for future reference?

For me personally, I'm much more comfortable with Rust than I am with C/C++, so those are the issues *I'd* be most interested in.

**Comment by djmitche** [comment]
I just tagged a few issues already labeled `taskchampion` as `rust`.  A good one to start with would be #2879.
